docs/src/pages/[platform]/connected-components/authenticator/intro.android.mdx

```diff
@@ -1 +1,9 @@
-The `Authenticator` component adds complete [authentication flows](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android) to your application with minimal boilerplate.
\ No newline at end of file
+import { Flex, Image } from '@aws-amplify/ui-react';
+
+The `Authenticator` component adds complete [authentication flows](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android) to your application with minimal boilerplate.
+
+<Flex direction="row" justifyContent="space-between">
```

Comment:
Did you test this on Mobile? I don't think this will wrap. Can you test the following?
```suggestion
<Flex direction={{base: "column", "medium:"row"}} justifyContent="space-between">
```

Reply:
Discussed offline and it looks fine on mobile currently.